test/support/feature_helpers.ex

```diff
@@ -23,31 +19,7 @@ defmodule Community.FeatureHelpers do
     submit_element({:css, "input[type=submit]"})
   end
 
-  def find_role(role_name) do
-    find_all_elements(:css, "[data-role=#{role_name}]")
-  end
-
-  def find_role(element, role_name) do
-    find_within_element(element, :css, "[data-role=#{role_name}]")
-  end
-
-  def text_for_role(role_name) do
-    role_name
-    |> find_role
-    |> show_text_for_all_roles(role_name)
-  end
-
-  def show_text_for_all_roles(elements, _) when length(elements) > 0 do
-    text_for_elements(elements)
-  end
-
-  def text_for_elements(elements) when length(elements) == 0 do
-    ""
-  end
-
-  def text_for_elements(elements) do
-    elements
-    |> Enum.map(&visible_text(&1))
-    |> Enum.join(" ")
+  def ignore_confirm_dialog do
+    execute_script("window.confirm = function(){return true;}")
```

Comment:
This is good to know! I'm sure this will come in handy in some of our other projects!